tests/integration_test/run_integration_test.py

```diff
@@ -0,0 +1,142 @@
+for filepath in file_paths:
+    graphs = load_graph_list_from_filepath(filepath)
+    filename = (filepath.split("/")[-1]).split(".")[0]
+    solution_map = {}
+    if filename in solution:
+        solution_map = solution[filename]
+
+    for i in range(0, len(graphs) - 1):
+        for j in range(i + 1, len(graphs)):
+            G = graphs[i]
+            H = graphs[j]
+
+            if console_pass or console_fail:
+                print('---------------------------')
+                print("Statistics of " + filename + "-" + str(i) + "_" + str(j) + ":")
+                print('---------------------------')
+
+            start_isomorph = time()
+            are_isomorph_actual = are_isomorph(G.copy(), H.copy())
+            end_isomorph = time()
+
+            start_amount_isomorphisms = time()
+            amount_isomorph_actual = amount_of_isomorphisms(G, H)
+            end_amount_isomorphisms = time()
+
+            # If solution of file is in solutions.py, check if solution is correct
+            if bool(solution_map):
+                graph_count += 1
+                are_isomorph_expected = (i, j) in solution_map
+                if are_isomorph_expected:
+                    amount_isomorph_expected = solution_map[(i, j)]
+                else:
+                    amount_isomorph_expected = 0
+                are_isomorph_result = True
+                amount_isomorph_result = True
+                # Graphs should be isomorphic
+                if are_isomorph_expected:
+                    if not are_isomorph_actual:
+                        if console_fail:
+                            fail("[FAIL] Graphs are isomorph, is_isomorph(G, H) did not detect it")
+                        error_count += 1
+                        are_isomorph_result = False
+                    else:
+                        if console_pass:
+                            passed("Graphs are isomorph")
+
+                    # If amount of isomorphisms in solution is -1, there is no answer known
+                    if amount_isomorph_expected != -1:
+                        if amount_isomorph_actual != amount_isomorph_expected:
+                            if console_fail:
+                                fail("[FAIL] Amount of isomorphisms should be " + str(amount_isomorph_expected) + ", not " + str(amount_isomorph_actual))
+                            error_count += 1
+                            amount_isomorph_result = False
+                        else:
+                            if console_pass:
+                                passed("Amount of isomorphisms is: " + str(amount_isomorph_actual))
+                # Graphs should not be isomorphic
+                else:
+                    if are_isomorph_actual:
+                        if console_fail:
+                            fail("[FAIL] Graphs are not isomorph, is_isomorph determined they were")
+                        error_count += 1
+                        are_isomorph_result = False
```

Comment:
Ik vind deze naamgeving een beetje verwarrend. Met result bedoel je volgens mij of de test passed of niet, en niet het resultaat van de test, dus of hij wel of niet isomorf is. Ik zou dit are_isomorph_passed ofzo noemen. Zelfde geldt voor amount_isomorph_result.